src/app/home/home.component.html

```diff
@@ -3,7 +3,7 @@
     <div class="pokeball-wrapper">
       <img class="pokeball" src="pokeball.png" alt="Pokeball" />
 
-      <a routerLink="display">
+      <a routerLink="/pokemons" [queryParams]="{ limit: 12, page: 0 }">
```

Comment:
It's good to never hardcode Values like this (12, 0).

Better to use a constant.